### PR TITLE
[IMP] website: rework badly worded instructions

### PR DESCRIPTION
**Pull request by ksc-odoo**
Made some grammatical and structure fixes throughout this documentation.

### CODE REVIEW

File: website/publish/domain_name.rst
Comment (AntoineVDV) [inline website/publish/domain_name.rst:91]: Please do as per https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_guidelines.html#use-single-underscore-suffixes-for-hyperlink-references

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hi, could you please rework your commit titles a little bit?
Ideally, they should follow this structure: "[TAG] path: a sentence that nicely follows 'If merged, this commit will...' " and be shorter than 80 characters. For example, "[IMP] website: rework badly worded instructions" could do the trick for this commit.
See [commit-template.txt](https://github.com/odoo/documentation-user/blob/14.0/commit_template.txt) for more details.

Also, your second commit only reverts a change that you introduce in your first commit. They should therefore be squashed.

**Comment by jcs-odoo** [comment]
@AntoineVDV 
Considering this doc is the same in 13.0 and 12.0, shouldn't this be based on another branch? At least 13.0?

**Comment by AntoineVDV** [comment]
> Considering this doc is the same in 13.0 and 12.0, shouldn't this be based on another branch? At least 13.0?

Since your team's opinion is shifting regularly, I don't comment anymore on whichever branch an app PR should be targeted, but you know my point of view on the matter :)

File: website/publish/domain_name.rst
Comment (ksc-odoo) [reply website/publish/domain_name.rst]: I believe I have fixed the single underscore issue.

**Comment by ksc-odoo** [comment]
your second commit only reverts a change that you introduce in your first commit. They should therefore be squashed.
> 
It currently says that "Merging is blocked" at the bottom of this page. How can I resolve that issue? Also, how do I "squash" two commits like this? Any insight would be greatly appreciated. Thanks! 

Also, sorry for all the errors. This is my first time doing anything like this, but I'm sure I'll figure it out. Thanks for your feedback and patience.

**Comment by AntoineVDV** [comment]
> Although, it might be difficult for KC to rebase all his numerous branches, now... I'm not sure what would be the easiest way to do so... If appliable.

If there are no conflicts, that shouldn't be too complicated.
What we want here is to apply this branch commit onto 13.0 rather than 14.0. A solution (which is not the shorted, but it the clearest IMO) could be to:
1. Prepare the branch with a single clean commit.
2. Use `git show` to show the latest commit of the current branch and take note of its hash. (`ec5cbe435b46566d40e4c1cfd607e00bde7f103d`, for example, is currently the latest commit of this branch.)
3. Use `git reset --hard 13.0` to reset the branch to be an exact copy of the locally stored 13.0.
4. Use `git cherry-pick commit_hash` to apply the commit onto 13.0's commit history.
5. Use `git push --force-with-lease` to replace the remote branch commits with the locale ones.
6. From GitHub interface, retarget the PR to 13.0.

**Comment by AntoineVDV** [comment]
> It currently says that "Merging is blocked" at the bottom of this page. How can I resolve that issue? Also, how do I "squash" two commits like this? Any insight would be greatly appreciated. Thanks!

The merge button is only green when there is no conflict when the base branch, and when there is at least one approving review (and no blocking review).
To merge/squash commits, do `git rebase -i 14.0` to interactively apply your commits again on 14.0.
The default text editor should open with content like this:
> pick abc123 [IMP] commit1
> pick abc456 [IMP] commit2

Simply replace 'pick' by 'fixup' on the second line to merge commit2 into commit1 and save the editor, then exit it.
If all goes fine, you should end up with a single commit containing the changes of both commits.

-------
> Also, sorry for all the errors. This is my first time doing anything like this, but I'm sure I'll figure it out. Thanks for your feedback and patience.

Don't worry about this, Git is not hard but takes some time to learn ;)

**Comment by jcs-odoo** [comment]
@ksc-odoo I would happily assist you to do these technical changes, now that I have the explanation from Antoine, above.
However, I'll let your reviewer @Lau-Odoo  check all your pull requests. It would be great if you could check which changes are appliable to 13.0 so we can do that together.

I'll probably be assigned to do the final review afterward, anyway.

Technically, once Ludvig or Antoine approved this Pull Request on GitHub, the merging button will be available to you. But in our team, we wait for the final-reviewer to approve as well before merging. Except for very small changes such as fixing a typo or an image: that can be done directly.

Cheers,
Jonathan

**Comment by jcs-odoo** [comment]
I put this PR back as a draft since it isn't ready to be merged.

**Comment by ksc-odoo** [comment]
> I put this PR back as a draft since it isn't ready to be merged.

Is there something I need to fix on my end?

**Pull request closed without merging**
